aws_xray_sdk/core/lambda_launcher.py

```diff
@@ -2,11 +2,11 @@
 import logging
 import threading
 
+import aws_xray_sdk
```

Comment:
This import is too general. It should be sth like `from aws_xray_sdk import global_sdk_config`